doc-content/enterprise-only/openshift/additional-server-managed-deploy-assy.adoc

```diff
@@ -9,7 +9,7 @@ ifdef::context[:parent-context: {context}]
 
 You can deploy an additional managed {KIE_SERVER} to an authoring or managed environment.  Deploy the server in the same project as the {LOCAL_CENTRAL} deployment.
 
-In order for the {LOCAL_CENTRAL} to connect to the {KIE_SERVER}, you must enable the `OpenShiftStartupStrategy` setting for your environment. For instructions abuot enabling the `OpenShiftStartupStrategy` setting, see <<startupstrategy-enable-proc>>.
+If you deployed a single authoring environment using the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring.yaml` template, in order for the {LOCAL_CENTRAL} to connect to the {KIE_SERVER} you must enable the `OpenShiftStartupStrategy` setting for your environment. For instructions about enabling the `OpenShiftStartupStrategy` setting, see <<startupstrategy-enable-proc>>. You do not need to complete this procedure for a high-availability authoring environment.
```

Comment:
The sentence order read a little confusing to me. If this rewrite retains your point, I suggest making the change:

If you deployed a single authoring environment using the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring.yaml` template, you must enable the `OpenShiftStartupStrategy` setting for your environment for the {LOCAL_CENTRAL} to connect to the {KIE_SERVER}. For instructions about enabling the `OpenShiftStartupStrategy` setting, see <<startupstrategy-enable-proc>>. You do not need to complete this procedure for a high-availability authoring environment.